py/dml/ctree.py

```diff
@@ -4610,7 +4610,7 @@ def log_statement(site, node, indices, logtype, level, groups, fmt, *args):
         groups = mkIntegerLiteral(site, 0)
 
     inargtypes = (([TInt(32, True)] if lvl else [])
-                  + [TPtr(TNamed("conf_object_t")), TInt(32, True),
+                  + [TPtr(TNamed("conf_object_t")), TInt(64, False),
```

Comment:
What's with this change?

Reply:
This is from the 'random cleanups' commit. uint64 is now a better approximation than uint32 of the loggroup arg of `SIM_LOG_*`. I don't think this change makes a difference to the behaviour, it's just for conciseness.